src/store/index.js

```diff
@@ -188,17 +177,33 @@ export const store = new Vuex.Store({
                 
                 // Create Array with sleep data for each weekday for chart
                 this.state.weeklyEntriesSleep = this.state.entries[0].sleep.map(a => a.hrsSlept);
+
+                // Show weekday entry green if user slept more than 7hrs, red if less
+                this.state.chartColors = [];
+                this.state.weeklyEntriesSleep.forEach(entry => {
+                    if (entry > 7) {
+                        this.state.chartColors.push('rgb(175, 228, 107)')
+                    } else {
+                        this.state.chartColors.push('#f87979')
+                    }
+                });
             })
             firebase.firestore().collection('needs/2/social').where("timeStamp", ">", from_date).orderBy("timeStamp", "desc").get().then((querySnapshot)=>{
                 querySnapshot.forEach(doc=>{
                     this.state.entries[1].social.push(doc.data())
                 })
             });
+            
             firebase.firestore().collection('users').get().then((querySnapshot)=>{
                 querySnapshot.forEach(doc=>{
                     this.state.users.push(doc.data())
                 })
+
+                if (this.state.needs[0].status != 0) {
```

Comment:
Depending on your use-case here, you might prefer to use stricter !== comparison (see here: https://stackoverflow.com/questions/8616483/javascript-comparison-operators-vs)